database/migrate.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+const postgrator = require('postgrator')
+const path = require('path')
+const minimist = require('minimist')
+const config = require('./../src/lib/config')
+
+postgrator.setConfig({
+  migrationDirectory: path.join(__dirname, '/migrations'),
+  schemaTable: 'schemaversion', // optional. default is 'schemaversion'
+  driver: 'pg', // or mysql, mssql
+  host: config.get('pgdb.host', '127.0.0.1'),
```

Comment:
You can specify a specific port already with env variables: in this specific case youll need to use `LABS_AUTH_SERVICE_pgdb_host=...` or `LABS_AUTH_SERVICE_pgdb_port=...`